### ISSUE TITLE
CREATE USER query fails if user already exists

### ISSUE THREAD

**Issue by bbinet** [influxdata/influxdb#5835]
As stated on https://github.com/influxdata/influxdb/pull/5360#issuecomment-185459635 the `if not exists` behavior should now be the default for all create statements, but when I try to create a user that already exists with influxdb nightly build (version 0.11.0.n1456300834, branch master, commit e32e5ff481c63f75d8f8bfe3f55dfc7a23559cf0, built 2016-02-24T08:01:27.779046), it does not work:
```
> CREATE USER u_a WITH PASSWORD 'u_a'
[query] 2016/02/24 13:32:37 CREATE USER u_a WITH PASSWORD [REDACTED]
[http] 2016/02/24 13:32:37 127.0.0.1 - root [24/Feb/2016:13:32:37 +0000] GET /query?db=_internal&epoch=ns&q=CREATE+USER+[REDACTED]+WITH+PASSWORD+%27[REDACTED]%27 HTTP/1.1 200 68 - InfluxDBShell/0.11.0.n1456300834 11a9392a-dafb-11e5-800d-000000000000 140.748357ms
ERR: user already exists
```

**Comment by bbinet** [comment]
I've just noticed the `category/clustering` label, but I think this issue is not related to clustering since the `CREATE USER` query failed for a standalone InfluxDB.

**Comment by e-dard** [comment]
FIxed via #6010
